webapp/components/integrations/components/installed_outgoing_webhooks.jsx

```diff
@@ -115,6 +115,7 @@ export default class InstalledOutgoingWebhooks extends React.Component {
                     creator={this.state.users[outgoingWebhook.creator_id] || {}}
                     canChange={canChange}
                     team={this.props.team}
+                    channel={channel}
```

Comment:
If this makes sense then https://github.com/mattermost/platform/issues/6454 commit should resolve this PR.